README.md

```diff
@@ -4,11 +4,18 @@ This is a simple webpage hosted on AWS Lambda using go templ + HTMX. It suggests
 
 See this hosted at https://subway.davidg.xyz
 
-## Build Instructions
-Run `./bundle.sh` to regenerate templates, bundle lambda handlers, and compile CSS. 
+## Build and Deploy Instructions
+Run `./bundle.sh` to regenerate templates, bundle lambda handlers, and compile
+CSS with tailwind. 
 
-Afterwards, you have to manually deploy the handlers to Lambda, and you need to manually drop `style.css` to the (hardcoded) S3 bucket. Yes, it's painful :)
+Deploy by `cd`-ing into the `cdk/` directory and running `cdk deploy`. Note
```

Comment:
It's pretty awkward that I can't do things from the project root. I might want to look into sorting that out later.